branch-4.0: [refactor](insert) unified INSERT_JOB to INSERT for EtlJobType (#56780)

sollhui opened the pull request:
### What problem does this PR solve?

pick https://github.com/apache/doris/pull/56780

NSERT_JOB was introduced in https://github.com/apache/doris/pull/26146, the purpose is to specify the job id for insert job. But there is no need to add a new job type, which would make the code difficult to understand. This pr aim to unified INSERT_JOB to INSERT.

### Release note

None

### Check List (For Author)

- Test <!-- At least one of them must be included. -->
    - [ ] Regression test
    - [ ] Unit Test
    - [ ] Manual test (add detailed scripts or steps below)
    - [ ] No need to test or manual test. Explain why:
        - [ ] This is a refactor/code format and no logic has been changed.
        - [ ] Previous test can cover this change.
        - [ ] No code files have been changed.
        - [ ] Other reason <!-- Add your reason?  -->

- Behavior changed:
    - [ ] No.
    - [ ] Yes. <!-- Explain the behavior change -->

- Does this need documentation?
    - [ ] No.
    - [ ] Yes. <!-- Add document PR link here. eg: https://github.com/apache/doris-website/pull/1214 -->

### Check List (For Reviewer who merge this PR)

- [ ] Confirm the release note
- [ ] Confirm test cases
- [ ] Confirm document
- [ ] Add branch pick label <!-- Add branch pick label that this PR should merge into -->

hello-stephen commented:
Thank you for your contribution to Apache Doris.
Don't know what should be done next? See [How to process your PR](https://cwiki.apache.org/confluence/display/DORIS/How+to+process+your+PR).

Please clearly describe your PR:
1. What problem was fixed (it's best to include specific error reporting information). How it was fixed.
2. Which behaviors were modified. What was the previous behavior, what is it now, why was it modified, and what possible impacts might there be.
3. What features were added. Why was this function added?
4. Which code was refactored and why was this part of the code refactored?
5. Which functions were optimized and what is the difference before and after the optimization?

sollhui commented:
run buildall

sollhui committed:
[refactor](insert) unified INSERT_JOB to INSERT for EtlJobType (#56780)

NSERT_JOB was introduced in https://github.com/apache/doris/pull/26146,
the purpose is to specify the job id for insert job. But there is no
need to add a new job type, which would make the code difficult to
understand. This pr aim to unified INSERT_JOB to INSERT.

sollhui commented:
run buildall
